cartography/intel/gcp/cloudfunction.py

```diff
@@ -79,7 +79,7 @@ def load_functions(neo4j_session: neo4j.Session,functions: List[Resource],projec
         :param neo4j session: The Neo4j session object
```

Comment:
We should avoid adding non sql related code in this PR. This may create confusion when we open PR to the Lyft branch.